widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -8873,4 +8906,13 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
 
 #refreshBrowserIcon:focus-visible {
   margin: var(--focus-ring-padding);
+}
+
+.fileUploadPanel-responsive > tbody > tr > td:nth-child(1) input {
+  flex: auto;
+}
+
+.fileUploadPanel-responsive > tbody > tr > td:nth-child(2),
+.fileUploadPanel-responsive > tbody > tr > td:nth-child(3)  {
+  flex: none;
```

Comment:
Where does this class come from? Why the `-responsive` suffix?
The file upload dialog will always be responsive from now on, for the Ruby theme.

Reply:
Just saw the new `fileUploadPanel-responsive` CSS classed being assigned in the related PR. Still, I maintain my comment about the choice of name.